### PR TITLE
add translations

### PR DESCRIPTION
**Pull request by DominikDoemsky**

### CODE REVIEW

File: lang/pt-PT.json [hunk 1]
```diff
@@ -2,13 +2,16 @@
...
-    "quilt-version": "Versão do carregador de colchas",
+    "quilt-version": "Versão Quilt loader",
     "plugin-load-problem": "O plugin '{{plugin-name}}' não pôde ser carregado.",
     "pocketmine-version": "Versão PocketMine",
     "bedrock-version": "Versão Bedrock",
     "forge-version": "Versão Forge",
+    "neoforge-version": "Versão NeoForge",
     "magma-version": "Versão Magma",
     "mohist-version": "Versão Mohist",
+    "arclight-version": "Versão ArClight",
```
Comment (pavog) [inline lang/pt-PT.json:13]: Arclight, not ArClight

File: lang/th.json [hunk 2]
```diff
@@ -7,20 +7,23 @@
     "pocketmine-version": "เวอร์ชั่น Pocketmine",
     "bedrock-version": "เวอร์ชั่น Bedrock",
     "forge-version": "เวอร์ชั่น Forge",
+    "neoforge-version": "เวอร์ชั่น Neoforge",
```
Comment (pavog) [inline lang/th.json:10]: Neo**F**orge

File: lang/th.json [hunk 3]
```diff
@@ -7,20 +7,23 @@
...
     "magma-version": "เวอร์ชั่น Magma",
-    "mohist-version": "เวอร์ชัน Mohist",
+    "mohist-version": "เวอร์ชั่น Mohist",
+    "arclight-version": "รุ่นอาร์คไลท์",
```
Comment (pavog) [inline lang/th.json:13]: They should not translate Arclight

File: lang/th.json [hunk 4]
```diff
@@ -7,20 +7,23 @@
...
+    "custom-skin-loader-version": "เวอร์ชันนักเทียบท่า Skin แบบกำหนดเอง",
```
Comment (pavog) [inline lang/th.json:14]: They should not translate CustomSkinLoader

File: lang/vi.json [hunk 5]
```diff
@@ -1,74 +1,88 @@
 {
-    "minecraft-version": "Phiên bản Minecraft",
+    "minecraft-version": "Phiên bản Minecraftjava và bedrock",
```
Comment (pavog) [inline lang/vi.json:2]: It should be "Minecraft version", not "Minecraft java and bedrock versions".

**Review by pavog — CHANGES_REQUESTED** [review]

**Review by pavog — CHANGES_REQUESTED** [review]
Some issues are not resolved yet. See fil, fr, gl, ...
Make sure to click "Load more..." right below "10 hidden conversations" ;-)

**Review by pavog — APPROVED** [review]

**Review by matthi4s — APPROVED** [review]

**Pull request merged**